apple/Tests/FerrostarCoreTests/__Snapshots__/ValhallaCoreTests/testValhallaRouteParsing.1.txt

```diff
@@ -80,6 +91,7 @@
         - spokenInstructions: 0 elements
         - visualInstructions: 0 elements
       ▿ RouteStep
+        - annotations: Optional<Array<String>>.none
```

Comment:
Manually redacting this until we solve the next ticket of deserializing all of this.